webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder.js

```diff
@@ -772,7 +772,12 @@ function showHideRunLogs(scrollTarget, init, refresh) {
 }
 
 function initDatePickers(id, action) {
+  var dateNow = new Date();
+  var twoHoursBefore = new Date();
+  twoHoursBefore.setHours(twoHoursBefore.getHours() - 2);
```

Comment:
Let me know if 2 hours windows is relevant or not

Reply:
It seems perfect to me!